db/migrate/20231003071434_add_browser_accept_language_to_users.rb

```diff
@@ -0,0 +1,6 @@
+class AddBrowserAcceptLanguageToUsers < ActiveRecord::Migration[7.0]
+  def change
+    add_column :users, :browser_accept_language, :string
+    add_column :users, :browser_accept_language_updated_at, :datetime
```

Comment:
While convenient, I'm trying to avoid increasing the size of the users' column, especially when the column may not directly be related to 'user'. I also see possibility of it changing base on user's devices or choice of browsers.